app/views/install/compose.phtml

```diff
@@ -727,7 +727,7 @@ services:
 
   openruntimes-executor:
     container_name: openruntimes-executor
-    hostname: appwrite-executor
+    hostname: exc1
```

Comment:
We might need to remove this change as it can affect people upgrading 🤔

Reply:
Should be fine as long as we update env var default [here](https://github.com/appwrite/appwrite/blob/sync-1.5.x-with-cloud/app/config/variables.php#L767-L774) as Steven mentioned

Reply:
Yea that will fix it for new installations of 1.5.7 but not for people upgrading from 1.5.6

Reply:
Because the variable has `overwrite` set to true, the value will be updated for upgrades too